ovs/v1/ovs.yaml

```diff
@@ -242,39 +242,15 @@ paths:
           application/json:
             schema:
               allOf:
-                - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/EVENT_DOMAIN/1.0.0#/components/schemas/transportEventBody'
-                - type: object
```

Comment:
Removed overwriting of facility Type, as this is set correct on OperationalEvent